Quadriga/src/main/webapp/WEB-INF/views/editing/networkvisual.jsp

```diff
@@ -34,7 +34,7 @@ $("input[type=button]").button().click(function(event) {
 
 <body
 	onload="d3visualize(<c:out value='${jsonstring}'></c:out>,<c:out value='${networkid}'></c:out>,<c:out value='"${pageContext.servletContext.contextPath}"'></c:out>,'force');" />
-
+<a href="#" onclick="goFullscreen('chart')"> <img src="${pageContext.servletContext.contextPath}/resources/fullscreen_icon.png" width="30" height="30"  style="float: left"/> </a>

```

Comment:
why don't you use font-awesome instead of an image? Easier to maintain and change.